website/pages/_document.js

```diff
@@ -9,7 +9,7 @@ export default class MyDocument extends Document {
 
   render() {
     return (
-      <html>
+      <Html>
```

Comment:
After upgrading `next` to `9.5.5.`, I started seeing this warning:

<img width="896" alt="consul-custom-html-error" src="https://user-images.githubusercontent.com/4624598/101926737-9295db00-3ba1-11eb-9de2-19e34a234bcf.png">

So I followed [NextJS's guide on fixing the error](https://github.com/vercel/next.js/blob/master/errors/missing-document-component.md).